sdk/spring/spring-test-template.yml

```diff
@@ -115,3 +115,44 @@ stages:
                 az logout
             condition: always()
             displayName: 'Delete On-demand test user'
+
+          - script: |
+              python --version
+              python -m pip install setuptools termcolor in_place
+              python sdk/spring/scripts/replace_util.py --module spring --log debug --color false
+            displayName: 'Run replace_util.py'
+          - task: Maven@3
+            displayName: 'Build and Install again, JDK Version: $(JavaBuildVersion)'
+            inputs:
+              mavenPomFile: pom.xml
+              goals: 'install'
+              options: '$(DefaultOptions) -DskipTests -Dgpg.skip -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true -pl $(ProjectList) -am'
+              mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
+              javaHomeOption: 'JDKVersion'
+              jdkVersionOption: $(JavaBuildVersion)
+              jdkArchitectureOption: 'x64'
+              publishJUnitResults: false
+
+          - script: |
+                az cloud set --name AzureChinaCloud
```

Comment:
For there are scirpts being scheduled twice, is it valuable that we move each of them to a file and call it then? But each scirpt only contains 2-3 lines of code....